docs/accessibility/guides/maintaining-accessibility.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Maintaining accessibility
+sidebar_position: 30
+---
+
+This section discusses the different mindset and success metrics you may have when a project moves from remediating known issues, to long term accessibility maintenance.
+
+## Switching from “improving” to “maintaining”
+
+As soon as a given accessibility rule reaches zero violations for a given page/component, or for the entire application, you enter “maintenance mode” for that rule. If the content being tested is stable, any time failure of that rule shows back up in the reports for that area on a feature branch, it will be clear that changes on that feature branch are the reason for the failure. These changes could be changes in the application code that made it less accessible, **or** changes in the tests to cover new parts of the application.
```

Comment:
```suggestion
As soon as a given accessibility rule reaches zero violations for a given page, component, or application, you enter “maintenance mode” for that rule. If the content being tested is stable, any time a failure of that rule shows back up in the reports for that area on a feature branch, it will be clear that changes on that feature branch are the reason for the failure. These changes could be changes in the application code that made it less accessible, **or** changes in the tests to cover new parts of the application.
```